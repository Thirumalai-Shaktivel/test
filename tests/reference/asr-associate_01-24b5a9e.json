{
    "basename": "asr-associate_01-24b5a9e",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/associate_01.f90",
    "infile_hash": "2c30bae981ba0fa05fb7f5715aeb34ee5a742af7c732b17902ba4ae0",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-associate_01-24b5a9e.stdout",
    "stdout_hash": "280335b90d0ea9eed4544cabd6496605ae32912a77b80248153e6cfb",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}