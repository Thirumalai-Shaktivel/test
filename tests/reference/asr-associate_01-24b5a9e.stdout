(TranslationUnit (SymbolTable 1 {associate_01: (Program (SymbolTable 2 {a: (Variable 2 a Local () Default (Real 4 [(() ())]) Source Public), a_1: (Variable 2 a_1 Local () Default (Real 4 []) Source Public), b: (Variable 2 b Local () Default (Real 4 [(() ()) (() ())]) Source Public), c: (Variable 2 c Local () Default (Real 4 [(() ()) (() ()) (() ())]) Source Public), c_234: (Variable 2 c_234 Local () Default (Real 4 []) Source Public), n: (Variable 2 n Local () Default (Integer 4 []) Source Public), x: (Variable 2 x Local () Default (RealPointer 4 [(() ())]) Source Public), y: (Variable 2 y Local () Default (RealPointer 4 [(() ()) (() ()) (() ())]) Source Public)}) associate_01 [(= (Var 2 n) (ConstantInteger 10 (Integer 4 []))) (Print () []) (Print () []) (Print () []) (= (Var 2 a_1) (ArrayRef 2 a [(() (ConstantInteger 1 (Integer 4 [])) ())] (Real 4 [(() ())]))) (= (Var 2 c_234) (ArrayRef 2 c [(() (ConstantInteger 2 (Integer 4 [])) ()) (() (ConstantInteger 3 (Integer 4 [])) ()) (() (ConstantInteger 4 (Integer 4 [])) ())] (Real 4 [(() ()) (() ()) (() ())]))) (Print () [(Var 2 a_1)]) (Print () [(Var 2 c_234)])])}) [])
